lib/rules/max-nesting-depth/index.mjs

```diff
@@ -124,7 +124,11 @@ const rule = (primary, secondaryOptions) => {
 			const normalized = parser().processSync(selector, { lossless: false });
 			const selectors = normalized.split(',');
 
-			return selectors.every((sel) => extractPseudoRule(sel));
+			const filteredSelectors = selectors.filter(
+				(sel) => !optionsMatches(secondaryOptions, 'ignoreRules', sel),
+			);
+
+			return filteredSelectors.every((sel) => extractPseudoRule(sel));
```

Comment:
```suggestion
			const matches = optionsMatches.bind(null, secondaryOptions, 'ignoreRules');

			return selectors.filter((s) => !matches(s))
					.every((s) => extractPseudoRule(s));
```

I don't know which one is better but that's an alternative.
It's a matter of taste I guess.

Reply:
Thank you for the suggestion. I appreciate your input, but I feel that the code you proposed might have added a bit more complexity.